labeled_data/technology/ai/generative_ai/application/index.yml

```diff
@@ -1,6 +1,13 @@
 name: Application
```

Comment:
The name of this label should be `Artificial Intelligence - Application`

Reply:
Yes, `Application` is not a complete name. However, it seems more accurate to use `Artificial Intelligence - Generative Artificial Intelligence - Application` instead of `Artificial Intelligence - Application` since the path is `:technology/ai/generative_ai/application/index.yml`. Is there a spelling error, or is my understanding of the pattern incorrect？

Reply:
You are right actually, my fault.

Reply:
OK~

Reply:
Another problem arose when I changed the name settings in yml files: Should I capital the first letter of all the categories in the name? The directories `ai` and `generative_ai` seem to be a special case to others.

Reply:
I think like in Database and Big Data labels, all the first letters are capitalized.

Reply:
OK, LGTM~